code-challenges.test.js

```diff
@@ -47,21 +97,65 @@ const studyMinutesWeek2 = {
   friday: 15,
   saturday: 65
 }
+expect(leastToGreatest(studyMinutesWeek1)).toEqual[15, 15, 20, 30, 30, 60, 90]
+expect(leastToGreatest(studyMinutesWeek2)).toEqual[10,15,20,45,60,65]
+})
+})
+
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
+// Expected output: [15, 15, 20, 30, 30, 60, 90]
 
 // b) Create the function that makes the test pass.
+// new function will correspond with describe function and do as it is named and return the expected result
+//empty array will bring in object values and sort it
+
+const leastToGreatest = (Object) => {
+emptyArr = Object.values(Object).sort((a,b) => a - b);
+return emptyArr
+}
```

Comment:
Good work! Make sure to run your code and copy and paste any errors you're getting.  While the logic is good, your syntax is off a bit.  Object with a capital O is a class, not a variable.  It's easy to get confused so be sure to double check your work and  look at error codes. :)